Automattic/jetpack
# Backup: Soft fail if using Jetpack and Jetpack older than 10

#### Changes proposed in this Pull Request:

* Outputs admin notice and ends execution if Jetpack is installed and that version is older than 10.0.

#### Jetpack product discussion


p1HpG7-cL2-p2#comment-48369

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* JN site with an older stable version of Jetpack (9.9.1, using wp cli `wp plugin install jetpack --version=9.9.1` helps)
* This branch.
* Activate both.
* See admin notice.
* Update Jetpack
* Backup loads.
